nameless/__init__.py

```diff
@@ -0,0 +1,10 @@
+import discord
+import bot
+
+class NamelessText(bot.Extension):
+    @bot.event("on_message")
+    async def replace_nameless(ctx, message):
+        """Deletes any messages containing nameless and replaces it with Nameless"""
+        if "nameless" in message.content.replace(" ", ""):
+            await message.channel.send("{}\n There, fixed that for ya".format(message.content.replace("nameless", "Nameless")))
+            await message.delete()
```

Comment:
It should leave the originally message

Reply:
that's what's inside the .format(). It uses the previous message after replacing nameless with Nameless

Reply:
It should still not delete the original

Reply:
So basically copy the message, then have the bot say the message with nameless fixed?

Reply:
Yes, just remove line 10

Reply:
Ah, sorry my bad. I miss understood then